integration-tests/micrometer-mp-metrics/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+quarkus.log.category."io.quarkus.micrometer".level=DEBUG
+quarkus.log.category."io.quarkus.micrometer.runtime.binder.vertx".level=INFO
+
+quarkus.log.category."io.quarkus.bootstrap".level=INFO
+quarkus.log.category."io.quarkus.netty".level=INFO
+quarkus.log.category."io.quarkus.resteasy".level=INFO
+
+quarkus.log.category."io.netty".level=INFO
+quarkus.log.category."io.restassured".level=INFO
+quarkus.log.category."org.apache".level=INFO
+quarkus.log.category."org.jboss.resteasy".level=INFO
+
+quarkus.micrometer.binder.mp-metrics.enabled=true
```

Comment:
Can the presence of the API be enough to enable it, or we need to be specific?

Reply:
This one needs to be specific because (for the foreseeable future), the smallrye API will be around. I could leave it as default on if the classes are found and just .. let that be a coping mechanism...

Reply:
I've adjusted this extension so that MP Metrics compat layer is on if MP Metrics API is present.